post/index.md

```diff
@@ -0,0 +1,295 @@
+                },
+            })
+        )
+
+// Response
+[{...}] // total length 1
+```
+
+### Get Data By Year
+
+```js
+  restClient
+        .request(
+            readItems("articles", {
+                filter: {
+                    _or: [
+                            {
+                                "year(published_date)": {
+                                        _eq: 2023,
+                                },
+                            },
+                            {
+                                "year(published_date)": {
+                                        _eq: 2022,
+                                },
+                            },
+                    ],
+                },
+            })
+        )
+// Response
+[{...},{...}] // Total count 2
+```
+
+### Get Data By Month & Year & search
+
+```js
+   restClient
+        .request(
+            readItems("articles", {
+                filter: {
+                    // Notice below _and -> logical operator
+                    _and: [
+                        {
+                            // Month
+                            _or: [
+                                {
+                                    "month(published_date)": {
+                                        // January
+                                        _eq: 1,
+                                    },
+                                },
+                                {
+                                    "month(published_date)": {
+                                        // Feb
+                                        _eq: 2,
+                                    },
+                                },
+                                {
+                                    "month(published_date)": {
+                                        // March
+                                        _eq: 3,
+                                    },
+                                },
+                            ],
+                        },
+                        {
+                            // Year
+                            _or: [
+                                {
+                                    "year(published_date)": {
+                                        _eq: 2023,
+                                    },
+                                },
+                                {
+                                    "year(published_date)": {
+                                        _eq: 2022,
+                                    },
+                                },
+                            ],
+                        },
+                        {
+                            // Title search
+                            title: {
+                                _contains: "gpt",
+                            },
+                        },
+                    ],
+                },
+            })
+        )
+
+//Response
+[{...},{...}] // total count 2
+```
+
+## Aggregation & Grouping
+
```

Comment:
I would like to see you write an introduction to Aggregation & Grouping here, not a straight copy from the docs. Perhaps in this context you can find an interesting way to explain it.